backend/app/adapter/sqldb/short_link.go

```diff
@@ -18,6 +18,34 @@ type ShortLinkSql struct {
 	db *sql.DB
 }
 
+// UpdateTwitterTags updates Twitter meta tags for a given short link.
```

Comment:
nit: the order of metatags is OpenGraph then Twitter (see backend/app/entity/metatag/metatag.go )

Reply:
@arberiii Sorry, didn't get what you mean. Could you please clarify? Thanks!

Reply:
I see

Reply:
If you check the order of metatags [here](https://github.com/short-d/short/blob/fix-809-part-4/backend/app/entity/metatag/metatag.go), first comes OpenGraph then Twitter. But in this PR we have UpdateTwitterTags first then UpdateOpenGraphTags

Reply:
@arberiii Oh, okey.